project/plugins.sbt

```diff
@@ -18,7 +18,7 @@
 addSbtPlugin("com.etsy" % "sbt-checkstyle-plugin" % "3.1.1")
 
 // sbt-checkstyle-plugin uses an old version of checkstyle. Match it to Maven's.
-libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "8.23"
+libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "8.25"
```

Comment:
Why it is an old version? Looks like it is very new version?

https://mvnrepository.com/artifact/com.puppycrawl.tools/checkstyle

Reply:
The description `sbt-checkstyle-plugin uses an old version` means that `sbt-checkstyle-plugin` depends on the old version and we need to override it here. :)

Reply:
I see. Thanks for clarifying.